pkgs/applications/audio/squeezelite/default.nix

```diff
@@ -67,7 +68,8 @@ stdenv.mkDerivation {
     ++ optional opusSupport "-DOPUS"
     ++ optional pulseSupport "-DPULSEAUDIO"
     ++ optional resampleSupport "-DRESAMPLE"
-    ++ optional sslSupport "-DUSE_SSL";
+    ++ optional sslSupport "-DUSE_SSL"
+    ++ optional gpioSupport "-DGPIO";
```

Comment:
Why not turn this always on without an option? It does not need an extra dependency and the binary size is not increasing significantly, right?

Reply:
Correct, no extra dependencies and minimal change in binary size. I've updated the PR to reflect this suggestion.